### DISCUSSION TITLE
app crashing on heroku

### DISCUSSION THREAD

**Discussion by Pol528** [eritislami/evobot#354] [General] [answered]
hello, the bot is working perfectly fine on my pc but it gets this error on Heroku.
I have checked and the api key is good
thanks in advance!
![unknown](https://user-images.githubusercontent.com/68276676/99914371-a8805080-2d05-11eb-8095-01d00cecac90.png)

**Comment by eritislami** [comment, marked as answer]
Check the readme, did you add config vars on heroku app?
